src/routes/(admin)/account/+layout.server.ts

```diff
@@ -12,7 +12,7 @@ export const load: PageServerLoad = async ({
 
   const { data: profile } = await supabase
     .from("profiles")
-    .select(`full_name, website, company_name`)
+    .select(`full_name, website, company_name, id, avatar_url, updated_at`)
```

Comment:
Should just be `select *` if we're now using the DB type? Will make maintenance easier. P2.

Reply:
Yep, good catch, changed to `select *`. I think the only real reason not do to this is if you store something sensitive on the user profile, which would go against the row-level security model anyway.